### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lkisoslive**
Just a suggestion. We can also go with "Open", "Run" or "Start application". 
I also changed "install" to "installation", let me know what you think.

### CODE REVIEW

**Review by j9t — CHANGES_REQUESTED** [review]
As these two sections are related, do you have a preference for something of a similar/consistent structure? One would now be a concept (“install”/“installation”), the other a call to action (“launch”).

File: README.md [hunk 1]
```diff
@@ -181,12 +181,12 @@ If you have not done so, install Ruby. Here’s a helpful guide on [how to best
...
 Once you have installed Ruby, clone this repository to your machine. Then, navigate to it using Terminal or your preferred command line interface. Follow the steps below to run the site from your machine. **If you’re on Windows, don’t forget to run your CLI as an admin.**
 
-### First time install
+### First time installation
 
 1. Run `npm install`
 2. Run `bundle install`
 
-### Start-up
+### Launch application
```
Comment (j9t) [inline README.md:189]: “Application” is pushing it a little… The command would start a local server that's running the website—“Launch server and website” may describe it but seems lengthy, too.

**Comment by lkisoslive** [comment]
How about "Run" in that case?

On Wed, Aug 3, 2022 at 7:13 PM Jens Oliver Meiert ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In README.md
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_LivePersonInc_developers-2Dcommunity_pull_1782-23discussion-5Fr936863283&d=DwMFaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=ShrVgFLLAuzR8snVbrQkmq_9bmUP0A_cY3KBwAkUQBA&m=I2y9s0_vaJU1-Yautjb7pBu574W1NlS925q0eGxt__Z8lCq9ulE9CPgervW3U5tJ&s=84T3rImCKdzyhDW8eGJRs9aqaGQ_SrjNVmWzUjAcGs8&e=>
> :
>
> >
>  1. Run `npm install`
>  2. Run `bundle install`
>
> -### Start-up
> +### Launch application
>
> “Application” is pushing it a little… The command would start a local
> server that's running the website—“Launch server and website” may describe
> it but seems lengthy, too.
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_LivePersonInc_developers-2Dcommunity_pull_1782-23pullrequestreview-2D1060608644&d=DwMFaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=ShrVgFLLAuzR8snVbrQkmq_9bmUP0A_cY3KBwAkUQBA&m=I2y9s0_vaJU1-Yautjb7pBu574W1NlS925q0eGxt__Z8lCq9ulE9CPgervW3U5tJ&s=c_HaZFPSwGn4H7Vh8Bn4gi9M60Q6HUrRg9IBYN010vE&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AU3MSSED62JFXMVK4IVULHLVXKLEHANCNFSM55PAYDBA&d=DwMFaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=ShrVgFLLAuzR8snVbrQkmq_9bmUP0A_cY3KBwAkUQBA&m=I2y9s0_vaJU1-Yautjb7pBu574W1NlS925q0eGxt__Z8lCq9ulE9CPgervW3U5tJ&s=8D2nuuHR_t2sj4Dab75w_R6W8LydoU5V2ZdWXCta3z8&e=>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

-- 
This message may contain confidential and/or privileged information. 
If 
you are not the addressee or authorized to receive this on behalf of the 
addressee you must not use, copy, disclose or take action based on this 
message or any information herein. 
If you have received this message in 
error, please advise the sender immediately by reply email and delete this 
message. Thank you.

**Comment by j9t** [comment]
> How about "Run" in that case?

You mean one heading “Install,” and the other “Run,” or what’s your thinking?

File: README.md (hunk 1 above)
Comment (lkisoslive) [reply README.md:189]: Maybe we can stick with "Open" or "Launch" and the specifics will be underneath anyway?

Comment (j9t) [reply README.md:189]: “Install” and “Launch”?

Comment (lkisoslive) [reply README.md:189]: I like "Install" and "Launch". I think keeping it vague here is okay and that there is an implication to installation that it is initial or the first time. @j9t Are we good to go with this?

**Review by lkisoslive — COMMENTED** [review]
I like "Install" and "Launch". I think keeping it vague here is okay and that there is an implication to installation that it is initial or the first time.

File: README.md (hunk 1 above)
Comment (j9t) [reply README.md:189]: Sure, let’s do this! If you can update the PR, I’ll approve.

**Review by j9t — APPROVED** [review]

**Review by lkisoslive — COMMENTED** [review]
Thank you

**Pull request merged**